pages.pt_BR/common/at.md

```diff
@@ -2,6 +2,7 @@
 
 > Ferramenta para o agendamento de comandos.
```

Comment:
Why this page is in the `common/` folder?

Reply:
There is a variant in `common/` and `linux/`.